src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowMonitoringSpec.groovy

```diff
@@ -207,7 +207,6 @@ and flowLatencyMonitoringReactions is disabled in featureToggle"() {
     }
 
     def cleanupSpec() {
-        islHelper.restoreIsls(islsToBreak)
         getDatabase().resetCosts(getTopology().isls)
```

Comment:
It seems like we can omit getDatabase().resetCosts(getTopology().isls) as the appropriate cleanup action is added  in the scope of breakIsl().